.github/workflows/site.yml

```diff
@@ -30,6 +30,36 @@ jobs:
       branch: ${{ steps.branch.outputs.ref }}
 
     steps:
+      - uses: khan/pull-request-comment-trigger@master
```

Comment:
Had to add it for every case as every time a single string is compared. There is no workaround I think because I tried with different conditions ( Using OR, commas, brackets ) but no result was there. Also this was pointed out in [Multiple trigger key check](https://github.com/Khan/pull-request-comment-trigger/issues/10) .

Reply:
Changed with GitHub so runs for all commands.